src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -389,8 +390,31 @@ class AddItemsPage extends Component {
    * @public
    */
   getLineItemsToBeSaved(lineItems) {
-    const lineItemsToBeAdded = _.filter(lineItems, item =>
+    const newItems = _.filter(lineItems, item =>
       !item.statusCode && item.quantityRequested && item.quantityRequested !== '0' && item.product);
+    let lineItemsToBeAdded = [];
+    _.forEach(newItems, (item) => {
+      const duplicatedItem = _.find(
+        lineItemsToBeAdded,
+        duplicate => _.get(duplicate, 'product.id') === _.get(item, 'product.id'),
+      );
+      if (duplicatedItem) {
+        const duplicatedItemIndex = _.findIndex(
```

Comment:
Do you need to do nearly the same find twice? Perhaps you can do just the find index?

Reply:
In line 410 duplicatedItem.quantityRequested is used. We also need index of this item and that's why we're finding it twice.

Reply:
Can't you use then `lineItemsToBeAdded[duplicatedItemIndex]`?

Reply:
Sure I can, nevermind. Thanks!